docs/notebooks/mappings.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mapped modules\n",
+    "\n",
+    "`xclim` leaves users the ability to generate their own modules as one-to-one mappings with the existing indice library. These mappings can help in reproducing existing indice libraries (such as ICCLIM), with the added benefit of CF-compliant metadata, multilingual metadata support, and optimized calculations using federated resources (using Dask). By wrapping functions with different thresholds from default values, any module can leverage the logic of indices to create their own custom module without having to rewrite indices.\n",
```

Comment:
```suggestion
    "`xclim` gives users the ability to generate their own modules as one-to-one mappings with the existing indices library. These mappings can help in emulating existing libraries (such as ICCLIM), with the added benefit of CF-compliant metadata, multilingual metadata support, and optimized calculations using federated resources (using Dask). This can be used for example to tailor existing indices with predefined thresholds without having to rewrite indices.\n",
```